src/components/Game/Game.js

```diff
@@ -65,7 +71,7 @@ class GameView extends Component {
             lettersInSlots: updatedLettersInSlots,
         }, () => {
             if (updatedLettersInSlots.length === WORD_LENGTH) {
-                this.checkResult();
+                dispatch(submitAnswer(this.joinLetters(updatedLettersInSlots)));
```

Comment:
are you planning on implementing submit button as specified in https://trello.com/c/l00usX4x/14-nowy-ux-ui?

Reply:
yes